clippy_lints/src/attrs.rs

```diff
@@ -810,6 +838,27 @@ fn check_nested_cfg(cx: &EarlyContext<'_>, items: &[NestedMetaItem]) {
     }
 }
 
+fn check_nested_misused_cfg(cx: &EarlyContext<'_>, items: &[NestedMetaItem]) {
+    for item in items.iter() {
+        if let NestedMetaItem::MetaItem(meta) = item {
+            if meta.has_name(sym!(features)) {
+                span_lint_and_sugg(
+                    cx,
+                    MAYBE_MISUSED_CFG,
+                    meta.span,
+                    "feature may misspelled as features",
+                    "use",
+                    "#[cfg(feature = ...)]".to_string(),
+                    Applicability::HasPlaceholders,
```

Comment:
You can likely make this `MaybeIncorrect` by calling `value_str` to get the name of the feature

Reply:
Thanks!